buildpack.toml

```
@@ -278,6 +278,20 @@ api = "0.7"
       type = "GPL-2.0 WITH Classpath-exception-2.0"
       uri = "https://openjdk.java.net/legal/gplv2+ce.html"
 
+  [[metadata.dependencies]]
+    cpes = ["cpe:2.3:a:oracle:graalvm:22.3.0:*:*:*:community:*:*:*", "cpe:2.3:a:oracle:jdk:21.0.0:*:*:*:*:*:*:*"]
```

oracle graalvm?

---

Yes, that's OK. The CPE should match up against what Oracle uses to publish CVEs.

---

```suggestion
    cpes = ["cpe:2.3:a:oracle:graalvm:23.0.0:*:*:*:community:*:*:*", "cpe:2.3:a:oracle:jdk:21.0.0:*:*:*:*:*:*:*"]
```

I'm not sure about the version number, but I'm going to suggest we go with 23 to match what Bellsoft is using. We could use 21, but I think that'll overlap with older versions of native-image. I guess we'll ultimately have to see what oracle does for future CVEs, perhaps they'll change the `graalvm` part to something else 🤷 .